packages/frenetic/frenetic.5.0.2/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "1.2"
+available: [ ocaml-version >= "4.05.0" ]
+version: "5.0.2"
+maintainer: "Steffen Smolka <smolka@cs.cornell.edu>"
+authors: "Arjun Guha <arjun@cs.umass.edu>, Nate Foster <jnfoster@cs.cornell.edu>, Steffen Smolka <smolka@cs.cornell.edu>"
+homepage: "http://frenetic-lang.org"
+bug-reports: "https://github.com/frenetic-lang/frenetic/issues"
+dev-repo: "https://github.com/frenetic-lang/frenetic.git"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+build-doc: ["jbuilder" "build" "@doc" "-p" name "-j" jobs]
+build-test: ["jbuilder" "runtest" "-p" name "-j" jobs]
+depends: [
+  "cppo"
```

Comment:
Same here. For reference please see the previous version: https://github.com/ocaml/opam-repository/blob/master/packages/frenetic/frenetic.5.0.0/opam